ddp-workspace/projects/ddp-sdk/src/lib/services/authentication/auth0Adapter.service.ts

```diff
@@ -140,6 +140,7 @@ export class Auth0AdapterService implements OnDestroy {
             ...(additionalParams && {
                 ...additionalParams
             }),
+            time_zone: Intl.DateTimeFormat().resolvedOptions().timeZone || null,
```

Comment:
Plumbing is in place to pass along timezone, but need help with how to actually get it...

Reply:
I think we can use `Intl.DateTimeFormat`, it has good browsers support - 94%
https://caniuse.com/#search=Intl.DateTimeFormat

Reply:
Another option is to use a polyfill:
https://github.com/formatjs/date-time-format-timezone
Might be a lighter weight solution if we can figure out how to load it conditionally (which I believe Angular 8 makes possible, but more research needed). So only the (very few) browsers that don't support it will load it.
But I am inclined to punt on momentjs or polyfill and consider timezone "unknown" or default to NY time or something like that until we have a way for user to explicitly set it via UI